packages/chap-lib/src/components/maps/Basemap.tsx

```diff
@@ -0,0 +1,21 @@
+import { useContext, useEffect } from "react";
+import { MapContext } from "./MapItem";
+
+const Basemap = () : any => {
+  const map : any = useContext(MapContext);
+
+  useEffect(() => {
+    map.addLayer(
+      map.createLayer({
+        type: "tileLayer",
+        url: "//cartodb-basemaps-{s}.global.ssl.fastly.net/light_all/{z}/{x}/{y}.png",
+        attribution:
+          '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>, &copy; <a href="http://cartodb.com/attributions">CartoDB</a>',
+      })
+    );
+  }, [map]);
+
+  return null;
+};
+
+export default Basemap;
```

Comment:
Fix linting issues so all checks pass

Reply:
Any specific linting issues you were thinking of in this file? Herman said there's probably many other linting issues that we should fix in one big bulk?

Reply:
I was thinking about the linting issues shown for the PR, here it seems to be a missing newline at the end. Not important.

Reply:
Ok, not sure how to check the PR linting issues, maybe for now i'll just skip and we can walk through the linting later, I see many files are missing newline at end if that's the requirement.